spec/models/match_spec.rb

```diff
@@ -0,0 +1,77 @@
+require "rails_helper"
+
+RSpec.describe Match, type: :model do
+  let(:teams) { create_list(:team_with_members, 2) }
+  let(:team_a) { teams[0] }
+  let(:team_b) { teams[1] }
+
+  describe "score" do
+    let(:match) { build(:match, team_one_score: 3) }
+    it "is present" do
+      expect(match.team_one_score).to eq(3)
```

Comment:
this test does not give us value. there should be a test of when the model should be valid ( not verify that the presence of an attribute when it's valid)